### ISSUE TITLE
Stargate Journe 0.6.9+ fails to load

### ISSUE THREAD

**Issue by jinks** [Povstalec/StargateJourney#54]
I tried updating SGJourney from 0.6.8_Hotfix to 0.6.10 but I get the following error on all versions past 0.6.8:
```
[01:44:21] [main/ERROR] [minecraft/RegistryDataLoader]: Registry loading errors:                                                                                                                                     
> Errors in registry minecraft:root:                                                                                                                                                                                 >> Errors in element minecraft:worldgen/template_pool:   
java.lang.IllegalStateException: Unbound values in registry ResourceKey[minecraft:root / minecraft:worldgen/template_pool]: [sgjourney:stargate_outpost/lantean_outpost_ocean_pool, sgjourney:stargate_pedestal/stargate_pedestal, sgjourney:stargate_pedestal/stargate_pedestal_badlands, sgjourney:stargate_pedestal/stargate_pedestal_chulak, sgjourney:stargate_pedestal/stargate_pedestal_deep_dark, sgjourney:stargate_pedestal/star
gate_pedestal_desert, sgjourney:stargate_pedestal/stargate_pedestal_jungle, sgjourney:stargate_pedestal/stargate_pedestal_mushroom, sgjourney:stargate_pedestal/stargate_pedestal_snow]                                      at TRANSFORMER/minecraft@1.20.1/net.minecraft.core.MappedRegistry.m_203521_(MappedRegistry.java:331)
        at TRANSFORMER/minecraft@1.20.1/net.minecraft.resources.RegistryDataLoader.m_257131_(RegistryDataLoader.java:69)                                  
        at java.base/java.lang.Iterable.forEach(Iterable.java:75)                                                                                                                                                            at TRANSFORMER/minecraft@1.20.1/net.minecraft.resources.RegistryDataLoader.m_247207_(RegistryDataLoader.java:65)                                                                                           
        at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.WorldLoader.m_246152_(WorldLoader.java:54)                                                                                                                      at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.WorldLoader.m_245736_(WorldLoader.java:58)                                                                                                              
        at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.WorldLoader.m_214362_(WorldLoader.java:31)                                                                                                              
        at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.Main.lambda$main$2(Main.java:167)
        at TRANSFORMER/minecraft@1.20.1/net.minecraft.Util.m_214652_(Util.java:777)
        at TRANSFORMER/minecraft@1.20.1/net.minecraft.Util.m_214679_(Util.java:772)
        at TRANSFORMER/minecraft@1.20.1/net.minecraft.server.Main.main(Main.java:166)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at MC-BOOTSTRAP/fmlloader@1.20.1-47.2.19/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111)
        at MC-BOOTSTRAP/fmlloader@1.20.1-47.2.19/net.minecraftforge.fml.loading.targets.CommonLaunchHandler.serverService(CommonLaunchHandler.java:103)
        at MC-BOOTSTRAP/fmlloader@1.20.1-47.2.19/net.minecraftforge.fml.loading.targets.CommonServerLaunchHandler.lambda$makeService$0(CommonServerLaunchHandler.java:27)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.run(Launcher.java:108)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.main(Launcher.java:78)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
        at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)

[01:44:21] [main/WARN] [minecraft/Main]: Failed to load datapacks, can't proceed with server load. You can either fix your datapacks or reset to vanilla with --safeMode                                             
java.util.concurrent.ExecutionException: java.lang.IllegalStateException: Failed to load registries due to above errors                                                                                                      at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396) ~[?:?] {re:mixin}                                                                                                            
        at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2073) ~[?:?] {re:mixin}                                                                                                                 
        at net.minecraft.server.Main.main(Main.java:195) ~[server-1.20.1-20230612.114412-srg.jar%23957!/:?] {re:mixin,re:classloading,pl:mixin:A}                                                                            at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}                                                                                                                            
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}                                                                                                                  at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}                                                                                                  
        at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {re:mixin}                                                                                                                                        
        at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.runTarget(CommonLaunchHandler.java:111) ~[fmlloader-1.20.1-47.2.19.jar%2369!/:?] {}
        at net.minecraftforge.fml.loading.targets.CommonLaunchHandler.serverService(CommonLaunchHandler.java:103) ~[fmlloader-1.20.1-47.2.19.jar%2369!/:?] {}
        at net.minecraftforge.fml.loading.targets.CommonServerLaunchHandler.lambda$makeService$0(CommonServerLaunchHandler.java:27) ~[fmlloader-1.20.1-47.2.19.jar%2369!/:?] {}
        at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) ~[modlauncher-10.0.9.jar%2355!/:?] {}
        at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) ~[modlauncher-10.0.9.jar%2355!/:?] {}
        at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) ~[modlauncher-10.0.9.jar%2355!/:?] {}
        at cpw.mods.modlauncher.Launcher.run(Launcher.java:108) ~[modlauncher-10.0.9.jar%2355!/:?] {}                                                                                                                
        at cpw.mods.modlauncher.Launcher.main(Launcher.java:78) ~[modlauncher-10.0.9.jar%2355!/:?] {}
        at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) ~[modlauncher-10.0.9.jar%2355!/:?] {}              
        at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) ~[modlauncher-10.0.9.jar%2355!/:?] {}                  
        at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141) ~[bootstraplauncher-1.1.2.jar:?] {}                                             
Caused by: java.lang.IllegalStateException: Failed to load registries due to above errors                                                                                                                            
        at net.minecraft.resources.RegistryDataLoader.m_247207_(RegistryDataLoader.java:77) ~[server-1.20.1-20230612.114412-srg.jar%23957!/:?] {re:classloading,pl:accesstransformer:B}
        at net.minecraft.server.WorldLoader.m_246152_(WorldLoader.java:54) ~[server-1.20.1-20230612.114412-srg.jar%23957!/:?] {re:mixin,xf:fml:libx:registry_load,re:classloading,xf:fml:libx:registry_load,pl:mixin:
APP:ldlib-common.mixins.json:WorldLoaderMixin,pl:mixin:A}                                               
        at net.minecraft.server.WorldLoader.m_245736_(WorldLoader.java:58) ~[server-1.20.1-20230612.114412-srg.jar%23957!/:?] {re:mixin,xf:fml:libx:registry_load,re:classloading,xf:fml:libx:registry_load,pl:mixin:
APP:ldlib-common.mixins.json:WorldLoaderMixin,pl:mixin:A}                                                                                                                                                            
        at net.minecraft.server.WorldLoader.m_214362_(WorldLoader.java:31) ~[server-1.20.1-20230612.114412-srg.jar%23957!/:?] {re:mixin,xf:fml:libx:registry_load,re:classloading,xf:fml:libx:registry_load,pl:mixin:
APP:ldlib-common.mixins.json:WorldLoaderMixin,pl:mixin:A}                                                                                                                                                            
        at net.minecraft.server.Main.lambda$main$2(Main.java:167) ~[server-1.20.1-20230612.114412-srg.jar%23957!/:?] {re:mixin,re:classloading,pl:mixin:A}
        at net.minecraft.Util.m_214652_(Util.java:777) ~[server-1.20.1-20230612.114412-srg.jar%23957!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.j
son:UtilMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.thread_priorities.UtilMixin,pl:mixin:APP:ftbchunks-common.mixins.json:UtilMixin,pl:mixin:APP:bettermineshafts.mixins.json:SuppressLogMixin,pl:mixin:A}
        at net.minecraft.Util.m_214679_(Util.java:772) ~[server-1.20.1-20230612.114412-srg.jar%23957!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:kubejs-common.mixins.j
son:UtilMixin,pl:mixin:APP:modernfix-common.mixins.json:perf.thread_priorities.UtilMixin,pl:mixin:APP:ftbchunks-common.mixins.json:UtilMixin,pl:mixin:APP:bettermineshafts.mixins.json:SuppressLogMixin,pl:mixin:A}
        at net.minecraft.server.Main.main(Main.java:166) ~[server-1.20.1-20230612.114412-srg.jar%23957!/:?] {re:mixin,re:classloading,pl:mixin:A}
        ... 15 more     
```

Pack is ATM9 0.2.35.

**Comment by Povstalec** [comment]
You'll need to update your Common Stargates Datapack

**Comment by jinks** [comment]
I don't have Common Stargates installed, but I think you put me on the right path.

ATM9 has a bunch of sgjourney JSON files in KubeJS, guess I'll have to sort this out by hand or wait until the pack gets around to it.

**Comment by Povstalec** [comment]
> I don't have Common Stargates installed, but I think you put me on the right path.

ATM9 has Common Stargates Datapack installed by default. But since it hasn't updated to the newer mod version, the datapack version is also old.

But yes, you do have it installed, even if it's done through KubeJS.

**Comment by jinks** [comment]
> ATM9 has Common Stargates Datapack installed by default. 

OK, let me rephrase: it's not installed as a pack. So I will have to compare whatever they put in KubeJS with the old version of Common Stargates and see what they messed about with.

Anyway, not *your* problem. I'll close this as fixed.

**Comment by Povstalec** [comment]
Alright, good luck. Hopefully it won't be a hard fix.
